tasks/PageIteratorTask.md

```diff
@@ -72,5 +72,23 @@ TypeScript
         }; 
         let pageIterator = new PageIterator(client, response, callback); 
         pageIterator.iterate(); 
+        
+        // Example of passing headers with a PageIterator
+        let response: PageCollection = await client.api("/me/events").headers({ "Prefer": 'outlook.timezone= "utc"' }).get(); 
+        let callback = (data) => { 
+            console.log(data); 
+            return true; 
+        }; 
+        var requestOptions = { headers: { "Prefer":  'outlook.timezone= "utc"' } };
+        let pageIterator = new PageIterator(client, response, callback, requestOptions); 
+        pageIterator.iterate(); 
 ```
-
+## Future Implementations
+    - Support 4th requirement mentioned in this file. The progress can be tracked as follows -
+    | SDK         | Implemenation Status |
```

Comment:
typo: Implemenation becomes Implementation

This should be the implementation status for the PageIterator options feature.